packages/react/index.js

```diff
@@ -72,6 +72,7 @@ export {
   SuspenseList,
   unstable_withSuspenseConfig,
   block,
+  useOpaqueIdentifier,
```

Comment:
Remove this until it's stable

Reply:
Never mind, this file is only used for tests. `index.stable.js` is the stable interface.